plugins/foreman_remote_execution/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: plugin_index
 title: Foreman Remote Execution Documentation
-versions: [0.1, 0.0]
+versions: [nightly, 0.1, 0.0]
```

Comment:
Are you sure you want to link it here?  Many of the installation instructions etc are not going to install the version that people expect.